extension/lib/settings_wrapper.js

```diff
@@ -11,17 +11,25 @@ if (window[namespace] === true) {
 
 // List of chrome storage keys.
 const settingKey = {
-	SPEED: "speed", // WPM
+	// WPM. If not set yet, default value of 260.
+	SPEED: "speed", 
+
+	// Tracker settings customize the appearance of the tracker.
+	// It is represented by the dictionary {trackerSettingKey : trackerSettingValue}
+	// All of the keys are mapped to a value.
 	TRACKER_CUSTOM: "tracker_custom",
+
+	// This is a map of all words user has ever read to the number of documents they have occured in. 
+	// Every time they read a new document, it adds one to each word in the dict, or adds the word to the dict
+	// initialized at 1.
 	// TODO: After testing and erased Ben's local storage, change this key to "documentFreq"
 	// to match rest of naming schema. 
 	DOCUMENT_FREQ: "termDocumentFreq",
+
+	// URLs that user has visited. set<String>
 	VISITED_URLS: "urls"
 };
 
-// Tracker settings customize the appearance of the tracker.
-// It is represented by the dictionary {trackerSettingKey : trackerSettingValue}
-// All of the keys are mapped to a value.
 const trackerSettingKey = {
 	KEYWORD: "keyword",
 	HIGHLIGHTER: "highlighter",
```

Comment:
Note: 
Only need Keyword, due to changes from New_UI recent merge. Just keep an eye out for that in the merge w master